### PR TITLE
Restored font-palette-vs-shorthand test

### PR DESCRIPTION
**Pull request by tursunova**
This _partially_ reverts commit 9125e441cf246bf7a352eca3e649f205c7003967.

### CODE REVIEW

File: css/css-fonts/font-palette-vs-shorthand.html [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+<div id=a>A</div>
+<div id=b>A</div>
+<div id=c>A</div>
+<div id=d>A</div>
+<script>
+test(function() {
+    let testElem = document.getElementById("a");
+    let computed = window.getComputedStyle(testElem);
+    assert_equals(computed.fontPalette, "dark");
+    assert_equals(computed.font, "");
```
Comment (drott) [inline css/css-fonts/font-palette-vs-shorthand.html:50]: Did you want to change these lines where font is "", if yes, please file a separate issue with the question about the serialization of font shorthand when font-palette is non-initiail.

File: css/css-fonts/font-palette-vs-shorthand.html [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    assert_not_equals(computed.font.indexOf("colr"), -1);
+    /* And there should be no trace of this: */
+    assert_equals(computed.font.indexOf("dark"), -1);
+});
+
+test(function() {
+    let testElem = document.getElementById("c");
+    let computed = window.getComputedStyle(testElem);
+    assert_equals(computed.fontPalette, "dark");
+    assert_equals(computed.font, "");
```
Comment (drott) [inline css/css-fonts/font-palette-vs-shorthand.html:71]: Ditto.

**Comment by drott** [comment]
> This reverts https://github.com/web-platform-tests/wpt/commit/9125e441cf246bf7a352eca3e649f205c7003967.

Nit: This _partially_ reverts...

File: css/css-fonts/font-palette-vs-shorthand.html (hunk 1 above)
Comment (tursunova) [reply css/css-fonts/font-palette-vs-shorthand.html:50]: Yes, I will file a separate issue for that, thanks!

**Comment by tursunova** [comment]
Reverted in #36932

**Pull request closed without merging**